lib/examples/ext_ctrls_test/main.rs

```diff
@@ -0,0 +1,29 @@
+use nix::fcntl::{open, OFlag};
+use nix::sys::stat::Mode;
+use std::fs::File;
+use std::os::unix::io::FromRawFd;
+use v4l2r::ioctl::*;
+
+const DEVICE_PATH: &str = "/dev/video2";
```

Comment:
Can we make this a command-line parameter?

Reply:
Of course we can. That's one of the reasons this series is a draft PR. Will address in the next iteration.